core/includes/common.inc

```diff
@@ -7248,7 +7248,7 @@ function backdrop_common_theme() {
       'variables' => array('url' => NULL, 'title' => NULL),
     ),
     'more_link' => array(
-      'variables' => array('url' => NULL, 'title' => NULL)
+      'variables' => array('url' => NULL, 'text' => NULL)
```

Comment:
This would be an API change. We can just leave the second parameter as "title" and change the call to theme('more_link') within Views.